content/en/account_management/teams/github.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Provision with GitHub
```

Comment:
Short title is good for the side nav, but a more descriptive title might be help the page be more self-contained (and consumable by AI).
```suggestion
title: Provision Datadog Teams with GitHub
```